R-package/tests/testthat/test_config.R

```diff
@@ -0,0 +1,13 @@
+context('Test global configuration')
+
+require(xgboost)
```

Comment:
this isn't necessary, and I think you can remove it. `testthat` will already handle loading `{xgboost}`, based on the commands you have in https://github.com/dmlc/xgboost/blob/master/R-package/tests/testthat.R

Reply:
Removed in my latest commit.